templates/agent.yaml

```diff
@@ -39,6 +39,29 @@ spec:
           imagePullPolicy: {{ .Values.image.agent.pullPolicy }}
           resources:
             {{- toYaml .Values.resources.agent | nindent 12 }}
+          {{- with .Values.proxy }}
+          {{- if and .enabled (or .user.secretName .password.secretName) }}
+          env:
+            {{- with .user }}
+            {{- if .secretName }}
+            - name: PROXY_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .secretName }}
+                  key: {{ .secretKey | default "PROXY_USERNAME" }}
+            {{- end }}
+            {{- end }}
+            {{- with .password }}
+            {{- if .secretName }}
+            - name: PROXY_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .secretName }}
+                  key: {{ .secretKey | default "PROXY_PASSWORD" }}
+            {{- end }}
+            {{- end }}
+          {{- end }}
+          {{- end }}
```

Comment:
Why is it set both here and in the ``secrets-nlweb.yaml‎`` file? it is duplicated, I think this one can be removed